src/Components/codeMirror/language-server/lsp-extension.ts

```diff
@@ -60,7 +60,6 @@ export function languageServerExtensions(
         }
       }
     }),
-    autocompletion(lspClient, filename),
```

Comment:
I'm not sure why, but removing this definitely was required to get snippets working. I guess it's replacing the autocompletion extension setup by the file above.

In a quick test, removing this did not seem to do any harm, context aware completion still seemed to work. Still, we should try and work out why this was originally here and if it's still strictly necessary - though it doesn't seem to be.